<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>606</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="layout_scene"/>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>270</width>
          <height>200</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Viewer</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>251</width>
            <height>151</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="wireframe_checkbox">
             <property name="text">
              <string>Wireframe</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="debug_button">
             <property name="text">
              <string>Debug</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_mesh">
   <property name="text">
    <string>Load mesh</string>
   </property>
  </action>
  <action name="actionSave_mesh">
   <property name="text">
    <string>Save mesh</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAzerty">
   <property name="text">
    <string>azerty</string>
   </property>
  </action>
  <action name="actionQwerty">
   <property name="text">
    <string>qwerty</string>
   </property>
  </action>
  <action name="actionSkeleton">
   <property name="text">
    <string>Skeleton</string>
   </property>
  </action>
  <action name="actionWeight">
   <property name="text">
    <string>Weight</string>
   </property>
  </action>
  <action name="actionNo_picking">
   <property name="text">
    <string>no picking</string>
   </property>
  </action>
  <action name="actionRemove_selected_mesh">
   <property name="text">
    <string>Remove (&amp;X)</string>
   </property>
  </action>
  <action name="action_Unselect">
   <property name="text">
    <string>&amp;Unselect</string>
   </property>
  </action>
  <action name="actionYafaray">
   <property name="text">
    <string>Yafaray</string>
   </property>
  </action>
  <action name="actionPovRay">
   <property name="text">
    <string>PovRay</string>
   </property>
  </action>
  <action name="actionLuxRender">
   <property name="text">
    <string>LuxRender</string>
   </property>
  </action>
  <action name="actionExport_sunflow">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionSettings_sunflow">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExport_indigo">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionSettings_indigo">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionLoad_scene">
   <property name="text">
    <string>Load scene</string>
   </property>
  </action>
  <action name="actionSave_scene">
   <property name="text">
    <string>Save scene</string>
   </property>
  </action>
  <action name="actionSkeleton_picking">
   <property name="text">
    <string>skeleton picking</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
